product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_template.yaml

```diff
@@ -421,6 +421,14 @@
           :required: false
           :display: :edit
           :data_type: :integer
+        :storage_profile_filter:
```

Comment:
Also, there are other dialog yaml files that need to be updated with this new field.  `manageiq/product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_clone_to_vm.yaml`
`manageiq/product/dialogs/miq_dialogs/miq_provision_vmware_dialogs_clone_to_template.yaml` (This is used when cloning a VM or Template to a Template.  Does storage profile still apply here?)